src/basic/special.h

```diff
@@ -82,6 +82,7 @@
 #define SPECIAL_QUOTAON_SERVICE "quotaon.service"
 #define SPECIAL_REMOUNT_FS_SERVICE "systemd-remount-fs.service"
 #define SPECIAL_VOLATILE_ROOT_SERVICE "systemd-volatile-root.service"
+#define SPECIAL_UDEVD_SERVICE "systemd-udevd.service"
```

Comment:
hmm, are you sure? one is a testcase for unescaping, where udevd is just an example. and the other is the name of a man page. In both cases we don't actually mean the special service, we either use it as an example string (where it could be any other string too) or referencing something that is not actually the service itself (though admittedly a related resource).

I think we should keep that separate, no?